drivers/cache/cache_xtheadcmo.c

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright (c) 2025 MASSDRIVER EI (massdriver.space)
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/init.h>
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/cache.h>
+#include <zephyr/logging/log.h>
+
+#define DT_DRV_COMPAT xuantie_xtheadcmo
+
+LOG_MODULE_REGISTER(cache_xtheadcmo, CONFIG_CACHE_LOG_LEVEL);
```

Comment:
This driver is incomplete and you are not explaining anything about this in the commit.
The way it is it won't be approved.

Reply:
added some infos.